### PR TITLE
Add push pull support

### PR DESCRIPTION
**Pull request by HeDo88TH**
Support PR for https://github.com/DroneDB/DroneDB/pull/195

### CODE REVIEW

**Comment by HeDo88TH** [comment]
Commit https://github.com/DroneDB/Registry/pull/129/commits/a6e3630cad37203b174a94f5c5f2ed399833e742 Closes #130

**Comment by HeDo88TH** [comment]
I don't know why CI is failing. It seems like that some ddb dependency is missing. Still investigating because it's green on my machine 👍

File: .github/workflows/dotnet-core.yml [hunk 1]
```diff
@@ -31,8 +31,8 @@ jobs:
           mkdir -p build
           cd build
           cmake -DBUILD_TESTING=OFF ..
-          make -j$(nproc) ddb
```
Comment (pierotofy) [inline .github/workflows/dotnet-core.yml:34]: This will slow down compilation substantially...

**Review by HeDo88TH — COMMENTED** [review]
Fixed in https://github.com/DroneDB/Registry/pull/129/commits/2a3d0a1338cd28e38dc10ae3952e0d5b5624b339 👍

**Comment by pierotofy** [comment]
`appsettings.json` was mistakenly committed, while the default config was missing the UploadPath key (it's now fixed)

**Comment by HeDo88TH** [comment]
> `appsettings.json` was mistakenly committed, while the default config was missing the UploadPath key (it's now fixed)

That's what happens after 10 hours of programming. Thanks for fixing 🙏

**Comment by HeDo88TH** [comment]
https://github.com/DroneDB/Registry/pull/129/commits/985db8284876075025b0a36ebd23cb2f0f14eba2 closes #135

**Comment by HeDo88TH** [comment]
https://github.com/DroneDB/Registry/pull/129/commits/ff0b9a4edab08fcb96aaca33ea78cac2f6c14c99 closes #133

**Comment by HeDo88TH** [comment]
https://github.com/DroneDB/Registry/pull/129/commits/494c6086bdab2714c0180ebe2995cbc5060ce043 closes #132

**Comment by HeDo88TH** [comment]
https://github.com/DroneDB/Registry/pull/129/commits/dfe66c9e18b75e10146b5e47d839135eebaf2a6a closes #137

**Pull request merged**